CUDADataFormats/Track/interface/PixelTrackHeterogeneous_t.h

```diff
@@ -1,31 +1,33 @@
-#ifndef CUDADataFormatsTrackTrackHeterogeneous_H
-#define CUDADataFormatsTrackTrackHeterogeneous_H
+#ifndef CUDADataFormatsTrackTrackHeterogeneous_t_H
+#define CUDADataFormatsTrackTrackHeterogeneous_t_H
```

Comment:
Since you rename the main class/struct to `TrackSoAHeterogeneousT`, please rename the file to `CUDADataFormats/Track/interface/TrackSoAHeterogeneousT.h` and update the include guard to
```suggestion
#ifndef CUDADataFormats_Track_TrackSoAHeterogeneousT_h
#define CUDADataFormats_Track_TrackSoAHeterogeneousT_h
```

Reply:
why all those training "T"?

Reply:
Actually, they were already present. We just kept them.

Reply:
They were introduced in 2019: https://github.com/cms-patatrack/cmssw/commit/e4918e38dc1e123c01fe5f8972f5d68d9fd0adce#diff-2541934e85a8f2093b439dbe5bb14605d491bdce80dff75106c93502d68634e9R14

Reply:
The changes were done.